.eslintrc.js

```diff
@@ -58,7 +58,10 @@ module.exports = {
     {
       files: ['**/*.ts', '**/*.tsx'],
       rules: {
-        '@typescript-eslint/no-unused-vars': 'error',
+        '@typescript-eslint/no-unused-vars': [
+          'error',
+          { caughtErrors: 'none' },
```

Comment:
What is the reasoning behind this change?

Reply:
An update to the default rules in `@typescript-eslint/no-unused-vars` caught about 50 instances in our application of `catch (e)` or `catch(err)` and treated them as unused vars. This setting undoes that

Reply:
Could we instead replace the instances of `catch (e) { ... }` and `catch(err) { ... }` with `catch { ... }` (i.e., `catch` with no parameter)? I feel like that is cleaner than disabling this eslint rule completely. Maybe others have opinions here, too.

Reply:
To clarify, this setting is confusingly named: `caughtErrors: none` refers to this specific try-catch pattern, and doesn't disable the rule entirely. This just changes the defaults back to what we had originally instituted. I'm certainly not opposed to refactoring away from the `catch (e)` pattern, but I don't think this eslint rule defaults update warrants it

Reply:
Gotcha!